tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -1126,8 +1131,8 @@ def test_keras_autolog_infers_model_signature_correctly_with_nparray(
 
 @pytest.mark.large
 @pytest.mark.skipif(
-    Version(tf.__version__) < Version("2.6.0"),
-    reason="TensorFlow autologging is not used for vanilla Keras models in Keras < 2.6.0",
```

Comment:
No reason to disable these tests in TF < 2.6.0. They aren't testing vanilla keras. They use `tf.keras`.

Reply:
It's possible that the author of these tests disabled them due to an import error caused by a bad test fixture that I'm addressing as part of this PR (https://github.com/mlflow/mlflow/pull/5652/files#r846734223)